geometry_msgs/Pose2D msg
---
float64 sig
